### PR TITLE
Add testing section to Readme

### PR DESCRIPTION
**Pull request by bigmontz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
 
 For a customised logging target, you can implement the above interface and pass an instance of that implementation to the `BoltLogger` field.
 
+## Developing
```
Comment (fbiville) [inline README.md]: How about using `CONTRIBUTING.md` for this instead?
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#about-contributing-guidelines

Comment (fbiville) [reply README.md]:
```suggestion
## Development
```

File: README.md [hunk 2]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+
+This section describe instructions to be used during the development phase of the driver.
```
Comment (fbiville) [inline README.md]:
```suggestion
This section describes instructions to use during the development phase of the driver. 
```

File: README.md [hunk 3]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
 
+## Developing
+
+This section describe instructions to be used during the development phase of the driver. 
+
+### Testing
+
+Tests **require** the latest [Testkit 4.3](https://github.com/neo4j-drivers/testkit/tree/4.3), Python3 and Docker.
+
+Testkit is needed to be cloned and configured to run against the Go Driver. Use the following steps to configure Testkit.
```
Comment (fbiville) [inline README.md]:
```suggestion
Testkit needs to be cloned and configured to run against the Go Driver. Use the following steps to configure it.
```

File: README.md [hunk 4]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+git clone https://github.com/neo4j-drivers/testkit.git
+```
+
+2. Under the Testkit folder, install the requirements.
+
+```
+pip3 install -r requirements.txt
+```
+
+3. Define some enviroment variables to configure Testkit
```
Comment (fbiville) [inline README.md]:
```suggestion
3. Define the following required environment variables
```

File: README.md [hunk 5]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+export TEST_DRIVER_REPO=<path for the root folder of driver repository>
+```
+
+To run test against against some Neo4j version:
+
+```
+python3 main.py
+```
+
+More details about how to use Teskit could be found on [its repository](https://github.com/neo4j-drivers/testkit/tree/4.3)
```
Comment (fbiville) [inline README.md]:
```suggestion
More details about how to use Teskit can be found on [its repository](https://github.com/neo4j-drivers/testkit/tree/4.3)
```

File: README.md [hunk 6]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+3. Define some enviroment variables to configure Testkit
+
+```
+export TEST_DRIVER_NAME=go
+export TEST_DRIVER_REPO=<path for the root folder of driver repository>
+```
+
+To run test against against some Neo4j version:
+
+```
```
Comment (fbiville) [inline README.md]:
```suggestion
```shell
```

File: README.md [hunk 7]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+
+2. Under the Testkit folder, install the requirements.
+
+```
+pip3 install -r requirements.txt
+```
+
+3. Define some enviroment variables to configure Testkit
+
+```
```
Comment (fbiville) [inline README.md]:
```suggestion
```shell
```

File: README.md [hunk 8]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+
+1. Clone the Testkit repository
+
+```
+git clone https://github.com/neo4j-drivers/testkit.git
+```
+
+2. Under the Testkit folder, install the requirements.
+
+```
```
Comment (fbiville) [inline README.md]:
```suggestion
```shell
```

File: README.md [hunk 9]
```diff
@@ -288,3 +288,39 @@ type BoltLogger interface {
...
+
+### Testing
+
+Tests **require** the latest [Testkit 4.3](https://github.com/neo4j-drivers/testkit/tree/4.3), Python3 and Docker.
+
+Testkit is needed to be cloned and configured to run against the Go Driver. Use the following steps to configure Testkit.
+
+1. Clone the Testkit repository
+
+```
```
Comment (fbiville) [inline README.md]:
```suggestion
```shell
```

**Review by fbiville — APPROVED** [review]
Just a few suggested changes here and there.
LGTM otherwise.

File: README.md (hunk 1 above)
Comment (bigmontz) [reply README.md]: All the other drivers have building and development instructions in the Readme. Doing it for the go driver makes it more consistent with the other repos.

**Pull request merged**